lib/ansible/modules/user.py

```diff
@@ -92,6 +92,9 @@
             - B(Linux/Unix/POSIX:) Enter the hashed password as the value.
             - See L(FAQ entry,https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-generate-encrypted-passwords-for-the-user-module)
               for details on various ways to generate the hash of a password.
+            - "WARNING: If the value does not match a valid hash format, it is written directly to C(/etc/shadow).
+              The module has no way to verify/validate the value.
+              This typically breaks authentication, but can be used intentionally to lock an account."
```

Comment:
This kind of makes it sound like it is a specific behavior to just write the value directly only when it's not a valid format.

I think a better comment, and one not needing a bit `WARNING` would just be that values are not validated, and used as is.